examples/mappings/mappingfunc_module.py

```diff
@@ -0,0 +1,22 @@
+"""Python module with mapping functions."""
+import numpy as np
+
+
+def formula(symbols):
+    """Convert a list of atomic symbols to a chemical formula."""
+    lst = symbols.tolist()
+    return "".join(f"{c}{lst.count(c)}" for c in set(lst))
+
+
+def norm(array, axis=-1):
+    """Returns the norm array along the given axis (default the last)."""
+    # Note that `array` is a Quantity object.  The returned value
+    # will also be a Quantity object with the same unit.  Hence, the
+    # unit is always handled explicitly.  This makes it possible for
+    # conversion function to change unit as well.
+    return np.sqrt(np.sum(array**2, axis=axis))
+
+
+def max(vector):
```

Comment:
I think `max()` is a very good name of this mapping function. 

There are no risk for confusion with the builtin `max()` function, since they reside in different namespaces:
`buildins.max()` and `mappingfunc_module.max()`.

Reply:
But isnæt this bad practice? What if someone wants to use the built in max in the same code further down? This is an example, so mayeb not that important. But at the same time this is where people see how we suggest to use our code. Changing it to maximum will not make it less understandabale and removed the problem.

Reply:
Updated requirements.txt such that the example works again.